src/main.cpp

```diff
@@ -133,14 +135,17 @@ int main( int argc, char* argv[] )
   G4RunManager* runManager = new G4RunManager();
 
   // Set up physics processes
-  G4VModularPhysicsList* physicsList = new QGSP_BERT_HP();
+  // G4VModularPhysicsList* physicsList = new QGSP_BERT_HP();
```

Comment:
Not related to crystal studies

Reply:
Moved to another MR: https://github.com/bwynneHEP/SimplePetScanner/pull/3